pkg/cmd/roachtest/roachtestutil/mixedversion/planner.go

```diff
@@ -100,6 +143,11 @@ const (
 	LastUpgradeStage
 	RunningUpgradeMigrationsStage
 	AfterUpgradeFinalizedStage
+
```

Comment:
Super Nit: small part of me still like these to be in separate const blocks. But a return is probably good enough separation.